app/views/follow_up_audit/_weaknesses_by_state_table.html.erb

```diff
@@ -47,7 +47,7 @@
     <p style="margin-left: 1em;">* <%= raw being_implemented_resume %></p>
   <% end %>
   <% if awaiting_resume.present? %>
-    <p style="margin-left: 1em;">** <%= raw awaiting_resume %></p>
+    <p style="margin-left: 1em;">  * <%= raw awaiting_resume %></p>
```

Comment:
¿Este no se fue en un "buscar y reemplazar en masa"?

Reply:
No, el doble ** es la sublista en textile y en markdown pasa a (doble espacio)*
Este review está hecho manual manual, lo único que  salio _seak and destroy_ fue el textilize_without... a markdown_without... :D

Reply:
Claro, pero acá no hay una llamada a ```markdown```, esos ```*``` se usan como una referencia ;)

Se generan en la linea 26 de este mismo archivo.